### PR TITLE
Redux: Add and remove from book store

### PR DESCRIPTION
**Pull request by MrOmachi**
In this mile stone, I have:

Connected Redux and React in my app
Configured Redux store using the configureStore() function.
Understood the concept of state management.
Used store, actions, and reducers in React.
Made use of useSelector and useDispacth functions provided in the 'react-redux' library

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,6 @@ To get a local copy up and running follow these simple example steps.
```
Comment (HAMMAS-SALEEM) [inline README.md:59]: ### Your File Structure
should

### Expected File Strucutre
Screenshot from 2022-09-20 02-46-03

- [x] Please make your file structure as shown in the expected file structure to organize your project so in future you won't face any problems while expanding your whole project :blush Kindly make your redux folder outside the components folder :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f574770523531743971623739534e464f74732f67697068792e676966

## Hey @MrOmachi :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitFlow
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Descriptive PR Title

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (MrOmachi) [reply README.md:59]: Okay, thanks for the detailed review.

Comment (MrOmachi) [reply README.md:59]: Directories Arrangements fixed

**Review by Reem-lab — APPROVED** [review]
# Hi @MrOmachi,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the file structure✅
📌 Good commit messages ✅
📌 Professional README ✅

## Optional suggestions
- [ OPTIONAL ] I recommend keeping the files for the compoenets for the form and the book and display book inside the components directory just keep the files that are related to the function of redux inside it 🚀✨🔥💪🏻


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**